Cargo.toml

```diff
@@ -1,150 +1,147 @@
 [workspace.package]
```

Comment:
Here, I removed the `default-features = false` for all dependencies don't have a `std` feature